script/openqa-clone-custom-git-refspec

```diff
@@ -105,7 +105,10 @@ clone_job() {
     if [[ -z "$testsuite" ]] || [[ -z "$needles_dir" ]] || [[ -z "$productdir" ]]; then
         local json_url=${host}/tests/${job}/file/vars.json
         local json_data
+        set +e
         json_data=$(eval "${curl_openqa} -s ${json_url}")
+        [ -z "$json_data" ] && echo 'openQA job with no valid JSON data encountered. Please select another job, e.g. previous job in same scenario' && exit 1
```

Comment:
maybe use `[[ -z ... ]]` instead to use the same style in all if statements?

Reply:
Actually my test approach was incomplete. I have changed this line now to check with `jq` if the document can be parsed as JSON. This way we can also cover 404 pages as returned by openQA when trying to access any not found document and not existant jobs.